src/test/java/save_and_restart_use_case/ReloadGameInteractorTest.java

```diff
@@ -0,0 +1,70 @@
+package save_and_restart_use_case;
+
+import CoreEntities.IO.DictionaryDataReaderGateway;
+import CoreEntities.IO.Presenter;
+import core_entities.game_parts.BoardFactory;
+import core_entities.game_parts.DefaultBoardFactory;
+import core_entities.game_parts.GameState;
+import default_reference_values.DefaultBoardDataAccessObject;
+import launch_new_game_use_case.CreateDictionaryDataAccessObject;
```

Comment:
alot of these imports can be just replaced with a "packages.*;" instead. not required.